core/src/main/java/cf/dmms/app/core/author/AuthorService.java

```diff
@@ -11,29 +13,27 @@ public AuthorService(AuthorRepository authorRepository) {
 		this.authorRepository = authorRepository;
 	}
 
-	public AuthorDto getAuthorById(Long id) {
-		Author author = authorRepository.getOne(id);
-		return AuthorDto.from(author);
+	public Author getAuthorById(Long id) {
+		return authorRepository.getOne(id);
 	}
 
-	public AuthorDto createAuthor(AuthorDto authorDto) {
-		Author author = new Author(authorDto.getName(), authorDto.getSurname());
-		author = authorRepository.save(author);
-		authorDto.setId(author.getId());
-
-		return authorDto;
+	public Author createAuthor(Author author) {
+		return authorRepository.save(author);
 	}
 
-	public AuthorDto updateAuthor(Long authorId, AuthorDto authorDto) {
+	public Author updateAuthor(Long authorId, Author changedAuthor) {
```

Comment:
We have here Author and his id. Can't we just override existing one with this changed one?